src/arfsdao.ts

```diff
@@ -744,7 +742,7 @@ export class ArFSDAO extends ArFSDAOAnonymous {
 		while (hasNextPage) {
 			const gqlQuery = buildQuery({
 				tags: [
-					{ name: 'Parent-Folder-Id', value: folderIDs },
+					{ name: 'Parent-Folder-Id', value: folderIDs.map(toString) },
```

Comment:
is this `toString`:
 - window.toString
 - this.toString
 - Object.prototype.toString
 - none of them (?
 
 let's maybe do the mapping with an arrow function

Reply:
Let's not leave it to chance 😬 2663b2b